clean_code/bfs_maze.py

```diff
@@ -0,0 +1,92 @@
+from maze import BoardIntegrityError
+
+
+class BFS_search():
+    def __init__(self, graph):
+        self.graph = graph
+        self.finish_junction = self.get_finish_junction()
+        self.start_junction = self.get_start_junction()    
+        self.temp = self.start_junction
```

Comment:
HELL not! never call your variable `temp`. I can only guess what is its purpose...